### PR TITLE
feat(loader): report component status to cluster manager

### PR DESCRIPTION
**Pull request by guangrui-cloudnatix**

### CODE REVIEW

File: deployments/loader/templates/clusterrolebinding.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
```
Comment (kkaneda) [inline deployments/loader/templates/clusterrolebinding.yaml:2]: `RoleBinding` should be sufficient?

File: deployments/loader/values.yaml [hunk 2]
```diff
@@ -73,6 +73,16 @@ runOnce: false
 # The following default values work if model-manager-server runs in the same namespace.
 modelManagerServerWorkerServiceAddr: model-manager-server-worker-service-grpc:8082
 
+componentStatusSender:
+  enable : true
```
Comment (kenji-cloudnatix) [inline deployments/loader/values.yaml:77]: Can you add comments to `enable` and `name`?

File: deployments/loader/values.yaml [hunk 3]
```diff
@@ -73,6 +73,16 @@ runOnce: false
...
+  name: model-manager-loader
+  # initialDelay is the time to wait before starting the sender.
+  initialDelay: 5m
```
Comment (kenji-cloudnatix) [inline deployments/loader/values.yaml:80]: How about shorting this, say `1m`?

File: deployments/loader/values.yaml [hunk 4]
```diff
@@ -73,6 +73,16 @@ runOnce: false
...
+  # The interval time to send the component status.
+  interval: 1h
```
Comment (kenji-cloudnatix) [inline deployments/loader/values.yaml:82]: I think this can be more frequent (e.g., 10m or 15m)

File: loader/cmd/run.go [hunk 5]
```diff
@@ -100,7 +95,20 @@ func run(ctx context.Context, c *config.Config) error {
...
-	return s.Run(ctx, c.ModelLoadInterval)
+	ss, err := cmstatus.NewBeaconSender(c.ComponentStatusSender, grpcOption(c), logger)
+	if err != nil {
+		return err
+	}
+
+	eg, ctx := errgroup.WithContext(ctx)
+	eg.Go(func() error { return s.Run(ctx, c.ModelLoadInterval) })
+	eg.Go(func() error {
+		if c.ComponentStatusSender.Enable {
```
Comment (kenji-cloudnatix) [inline loader/cmd/run.go:106]: Slightly prefer

```go
if c.ComponentStatusSender.Enable {
  eg.Go(func() error {
   ss.Run(ctx) 
   return  nil
 }
```

**Review by kenji-cloudnatix — APPROVED** [review]
LG!

**Review by kkaneda — APPROVED** [review]

**Pull request merged**